syft/frameworks/crypten/__init__.py

```diff
@@ -2,20 +2,27 @@
 import syft
 
 from syft.frameworks.crypten.context import run_party
+from syft.frameworks.crypten import utils
 
 import crypten.communicator as comm
 import crypten
 
 
 def load(tag: str, src: int, **kwargs):
+    if tag == "crypten_model":
+        worker_model = syft.local_worker.get_worker_from_rank(src)
+        results = syft.local_worker.find_or_request(tag, worker_model)
+        assert len(results) == 1
+        model = results[0].get()
+        return utils.onnx_to_crypten(model.serialized_model)
```

Comment:
can the de-serialization happens during the get?

Reply:
Hmm..it should matter? (currently I applied a ```get_copy``` to the get)

Reply:
It think it should not happen there because "get" should return an ```Object``` and adding this functionality to the ```get``` involves adding more framework dependent code to the ```get``` method.